oceannavigator/frontend/src/components/Map.jsx

```diff
@@ -412,6 +412,15 @@ export default class Map extends React.PureComponent {
             switch (feat.get("type")) {
               case "area": {
                 return [
+                  new olstyle.Style({
+                    stroke: new olstyle.Stroke({
+                      color: "#FFFFFF",
+                      width: 2,
+                    }),
+                    fill: new olstyle.Fill({
+                      color: "#FFFFFF00"
+                    })
+                  }),
```

Comment:
It doesn't appear that openlayer's provides a method to outline polygons so as I workaround I layered two Styles with different colours and sizes to get the same effect.